code-examples/php/usage-examples/find.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once __DIR__ . "/vendor/autoload.php";
+
+$uri = "mongodb://localhost:27017";
+
+$client = new MongoDB\Client($uri);
```

Comment:
The client construction here is also inconsistent with the other files.

Reply:
We use configurable URI in our examples: 

```suggestion
$client = new Client(getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
```